### PR TITLE
Update ChromeOS installation instructions

### PR DESCRIPTION
**Pull request by timsneath**
_Description of what this PR is changing or adding, and why:_

This PR significantly simplifies and updates the installation instructions for ChromeOS. It is tested on recent versions of the operating system that have simplified installation for developers and migrated Linux from beta. It removes optional instructions that assume a wide variety of Linux distributions (e.g. prerequisites that are known to be available on ChromeOS). It also reduces the emphasis on Android, since ChromeOS can also be used for web or Linux development. 

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Review by khanhnwin — APPROVED** [review]

File: src/get-started/install/_android-setup-chromeos.md [hunk 1]
```diff
@@ -8,20 +8,32 @@
...
 
  ```terminal
 $ flutter doctor --android-licenses
 ```
 
+### Deploy to your Chromebook
+
+To deploy apps directly to your Chromebook, you need to do the following:
+
+ 1. [Enable ADB][] in Settings. Note that this will require you to reboot your
```
Comment (sfshaza2) [inline src/get-started/install/_android-setup-chromeos.md:28]:
```suggestion
 1. [Enable ADB][] in Settings. Note that this requires you to reboot your
```

File: src/get-started/install/_chrome-setup-chromeos.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Configuring web app support
+
+On ChromeOS, you do development work in a Linux container.
+However, the Chrome browser itself is part of the
+parent ChromeOS operating system,
+and Flutter doesn't have a means to call it with the required parameters.
+
+This issue is tracked as <https://github.com/flutter/flutter/issues/121462>.
```
Comment (sfshaza2) [inline src/get-started/install/_chrome-setup-chromeos.md:8]:
```suggestion
Track this issue at [Issue 121462: Improve the web debugging experience on Chromebooks]({{site.github}}/flutter/flutter/issues/121462).
```

File: src/get-started/install/_chromeos-android-sdk-setup.md [hunk 3]
```diff
@@ -8,13 +8,13 @@ $ sudo apt install default-jre
...
-### Install the Android SDK's
+### Install the Android SDKs
 
 Download the [Android SDK tools][] and
 select the “Command Line Tools only” option.
 
 Drag and drop the downloaded zip into your Linux Files folder through the
-Chrome OS Files app. This moves it to the home directory, notated as
+ChromeOS Files app. This moves it to the home directory, notated as
 $TOOLS_PATH going forward (`~/`).
```
Comment (sfshaza2) [inline src/get-started/install/_chromeos-android-sdk-setup.md:18]:
```suggestion
ChromeOS Files app. This moves the file to the home directory,
and is referred to as $TOOLS_PATH going forward (`~/`).
```

File: src/get-started/install/chromeos.md [hunk 4]
```diff
@@ -11,101 +11,23 @@ next:
 
 ## System requirements
 
-To install and run Flutter, your development environment
-must meet these minimum requirements:
+To install and run Flutter on a Chromebook, your machine
+must have [Linux][] enabled from the Developers tab of Settings.
```
Comment (sfshaza2) [inline src/get-started/install/chromeos.md:15]:
```suggestion
must have [Linux][] enabled from the **Developers** tab of Settings.
```

File: src/get-started/install/chromeos.md [hunk 5]
```diff
@@ -11,101 +11,23 @@ next:
...
-  * `xz-utils`
-* **Shared libraries**: Flutter `test` command depends on
-  this library being available in your environment.
-  * `libGLU.so.1` - provided by mesa packages such as `libglu1-mesa` on
-     Ubuntu/Debian
+The amount of disk space required will vary
+depending on which target platforms you enable.
+However, we recommend that you increase the disk size for the Linux environment
+from the default of 10GB to 32GB or larger,
+to accommodate Android Studio and other tooling.
```
Comment (sfshaza2) [inline src/get-started/install/chromeos.md:21]:
```suggestion
The amount of disk space required varies
depending on which target platforms you enable.
We recommend that you increase the disk size for the
Linux environment from the default of 10GB to 32GB or larger,
to accommodate Android Studio and other tooling.
```

**Review by sfshaza2 — APPROVED** [review]
LGTM with a few changes. You are on fire, @timsneath!

**Pull request merged**